Formula/chain-bench.rb

```diff
@@ -0,0 +1,22 @@
+class ChainBench < Formula
+  desc "Software supply chain auditing tool based on CIS benchmark"
+  homepage "https://github.com/aquasecurity/chain-bench"
+  url "https://github.com/aquasecurity/chain-bench/archive/v0.1.3.tar.gz"
+  sha256 "46b63528ddd0e09b65539a69aefbf4229343e636854d0dd5a241870717843423"
+  license "Apache-2.0"
+  head "https://github.com/aquasecurity/chain-bench.git", branch: "main"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-s -w -X=main.version=#{version}"), "./cmd/chain-bench"
+  end
+
+  test do
+    assert_raises(BuildError) {
+      system "#{bin}/chain-bench scan"
+    }
```

Comment:
We don't want to expect a ruby error here. Please use `shell_output` and `assert_match`to assert some output here

Reply:
@SMillerDev Sorry for the multiple iterations, this time is it sufficient?